public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -275,9 +249,7 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
     ? 'Folder name can be updated in folder view'
     : 'Name space can be updated folder view';
```

Comment:
nit: This sentence is weird, should it be

`namespace can be updated on the folder view` and what exactly is the folder view?

Reply:
I'll remove this. The folder input has a link where you can edit, and the namespace has a description below the label, which should be enough